cmd/kube-scheduler/app/options/configfile.go

```diff
@@ -93,7 +93,7 @@ func LogOrWriteConfig(fileName string, cfg *config.KubeSchedulerConfiguration, c
 	}
 
 	if klog.V(2).Enabled() {
-		klog.InfoS("Using component config", "config", buf.String())
+		klog.Info("Using component config", "\n-------------------------Configuration File Contents Start Here---------------------- \n", buf.String(), "\n------------------------------------Configuration File Contents End Here---------------------------------\n")
```

Comment:
We would be giving up structured logging in this case. But I think it's better in this case.

@logicalhan WDYT?

Reply:
@serathius probably has a more informed opinion

Reply:
I think we should be able to do it using structured logging, however we might need a special formatter struct for this. 

cc @pohly for recommendation

Reply:
A formatter doesn't help at the moment. Strings as values in a key/value pair will always be quoted in text output, that cannot be suppressed. It's worthwhile to add that, though, therefore I have filed https://github.com/kubernetes/kubernetes/issues/106262.

In the meantime this PR can and should be merged as-is.